NameGame/NameGame/State/Middleware/StateLogger.swift

```diff
@@ -11,7 +11,8 @@ import ReSwift
 final class StateLogger: StoreSubscriber {
     
     func newState(state: AppState) {
-        print("GameState: \(String(describing: state.gameState))")
+//        print("GameState: \(String(describing: state.gameState))")print("GameState: \(String(describing: state.gameState))")
```

Comment:
How about instead of having the game state logging commented out, we add a mode to our logger.

```
enum LogLevel {
  case fullState
  case gameState
}

var logLevel: LogLevel = .fullState

fun newState(state: AppState {
  switch logLevel {
    ...
  }
}
```